### PR TITLE
Update to log4j 2.17.

### PR DESCRIPTION
**Pull request by loganaden**
This fixes https://nvd.nist.gov/vuln/detail/CVE-2021-45105

### CODE REVIEW

File: zuul-sample/dependencies.lock [hunk 1]
```diff
@@ -1209,4 +1209,4 @@
             "locked": "1.7.32"
         }
     }
-}
\ No newline at end of file
+}
```
Comment (argha-c) [inline zuul-sample/dependencies.lock:1212]: Is this intentional? Doesn't look like a line ending

**Comment by loganaden** [comment]
Sorry for this. Will fix it.

**Comment by argha-c** [comment]
Let's update this to `2.17.1` now that there's a release.

**Comment by loganaden** [comment]
Will work on a new patch. Thanks @argha-c

**Comment by argha-c** [comment]
Closed in favor of https://github.com/Netflix/zuul/pull/1169

**Pull request closed without merging**